docs/payload-migration/03-core-data-models.md

```diff
@@ -0,0 +1,480 @@
+# Chapter 3: Core Data Models
+
+[← Back to Index](./README.md)
+
+---
+
+## Priority Order
+
+| Priority | Collection | PostgreSQL Table | Status | Notes |
+|----------|-----------|------------------|--------|-------|
+| 1 | People | `people` | 🔲 Todo | Base collection for individuals |
+| 2 | DJs | `djs` | 🔲 Todo | References People collection |
+| 3 | Artists | `artists` | 🔲 Todo | Musicians/bands with many-to-many to People |
+| 4 | Venues | `venues` | 🔲 Todo | Concert locations |
+| 5 | Media | `media` | 🔲 Todo | Upload collection for images |
+| 6 | Ads | `ads` | 🔲 Todo | Sponsor/advertisement data |
+| 7 | Concerts | `concerts` | 🔲 Todo | References Artists and Venues |
+| 8 | Songs | `songs` | 🔲 Todo | Music catalog with Artist references |
+| 9 | Records | `records` | 🔲 Todo | Albums (used by CD of the Week) |
+| 10 | CdOfTheWeek | `cd_of_the_week` | 🔲 Todo | Album reviews referencing Records |
+| 11 | OnDemand | `on_demand` | 🔲 Todo | Audio content referencing Artists/DJs |
+| 12 | Shows | `shows` | 🔲 Todo | Schedule entries referencing DJs |
+| 13 | Posts | `posts` | 🔲 Todo | Content blocks (unified Story + CustomText) |
+| 14 | Top11Contests | `top11_contests` | 🔲 Todo | Weekly contest config |
+| 15 | Top11Results | `top11_results` | 🔲 Todo | Published weekly results |
+| 16 | Top11Votes | `top11_votes` | 🔲 Todo | User votes (PostgreSQL native, not NoSQL) |
+| 17 | YearEndPolls | `year_end_polls` | 🔲 Todo | Annual poll config |
+| 18 | YearEndPollCategories | `year_end_poll_categories` | 🔲 Todo | Poll categories |
+| 19 | YearEndPollVotes | `year_end_poll_votes` | 🔲 Todo | User votes (PostgreSQL native) |
+| 20 | MRMTournaments | `mrm_tournaments` | 🔲 Todo | Modern Rock Madness tournaments |
```

Comment:
We aren't going to use MRM as an abbreviation for Modern Rock Madness anywhere in our schema or our code.